src/nni_manager/training_service/local/gpuScheduler.ts

```diff
@@ -54,6 +54,9 @@ class GPUScheduler {
             } catch (error) {
                 this.log.error('Read GPU summary failed with error: ', error);
             }
+            if (this.gpuSummary.gpuCount === 0) {
+                throw new Error('GPU not available. Please check your CUDA configuration');
```

Comment:
If the gpu_metrics_collector update gpu_metric file slowly, will gpuSummary.gpuCount updated to a positive number after several detection?